### PR TITLE
Test End Problem

### PR DESCRIPTION
**Pull request by asif-khan17**
**Fixed Issues**

#TestEnd

**Files Changed**

test.component.ts

**Checks**

- [ ] Naming Conventions
- [ ] Server side code comments (proper English, grammar and no spelling mistakes)
- [ ] Client side code comments (proper English, grammar and no spelling mistakes)
- [ ] Unused variables, methods, blank spaces and name spaces. 
- [ ] Optimal Code and code formatting
- [ ] Commit History is proper, no merge is done. 
- [ ] Commit/pull request messages should be proper to justify your feature
- [ ] All test cases are executed provided by tester.

**Known Issues**

- issue 1
- issue 2

**Comments**

anything that you want to convey about this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/promact/trappist/444)

### CODE REVIEW

File: Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs [hunk 1]
```diff
@@ -511,11 +507,12 @@ public class TestConductRepository : ITestConductRepository
                             }
 
                             testAnswers.TestConduct = testConduct;
-                            await _dbContext.TestAnswers.AddAsync(testAnswers);
-                            await _dbContext.SaveChangesAsync();
+                            testAnswersList.Add(testAnswers);
                         }
-                        
                     }
+                    await _dbContext.TestAnswers.AddRangeAsync(testAnswersList);
```
Comment (rushi216) [inline Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs]: Why are you adding this two times?

Comment (asif-khan17) [reply Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs]: one TestAnswer table and other one is TestConduct table

Comment (rushi216) [reply Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs]: :smile: sunday mornings

Comment (rushi216) [reply Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs]: Rebase it

**Pull request merged**